atlasdb-feign/src/main/java/com/palantir/atlasdb/http/FeignOkHttpClients.java

```diff
@@ -89,17 +91,21 @@ private FeignOkHttpClients() {
      */
     public static <T> Client newOkHttpClient(
             Optional<SSLSocketFactory> sslSocketFactory,
+            Optional<ProxySelector> proxySelector,
```

Comment:
While technically a public API break this one might be okay, as no one uses it internally when I checked.